go/ai/document.go

```diff
@@ -11,6 +11,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
according to the guidelines `SPDX-License-Identifier` can be used __instead of__ the license header, but not both.